### DISCUSSION TITLE
What am i supposed to do with this? does this 2 mods doesn't works together?

### DISCUSSION THREAD

**Discussion by bubble13699** [shedaniel/RoughlyEnoughItems#906] [Q & A] [answered]
![Untitled](https://user-images.githubusercontent.com/101046132/173221852-3c835507-3d23-493b-9051-d8da1786cf0f.png)

**Comment by shedaniel** [comment, marked as answer]
You can ignore the warning and press continue at the bottom, this issue has to be fixed by BitsAndChisels.

**Reply by bubble13699** [reply]
done that 4 times already..but the game freeze right after i pressed continue.

**Reply by shedaniel** [reply]
Oh interesting, I will take a look, but the main issue should still be mitigated by ChiselsAndBits, is this 1.18 or 1.19?

**Reply by bubble13699** [reply]
i played on 1.19. 
so, right now i need to delete ChiselsAndBits in order for the game to run right?

**Reply by shedaniel** [reply]
I have created issue #912 for this. For now you might need to remove ChiselsAndBits, but I reckon I will make a fix soon.
